[2021-12-13 20:41:00,439] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag.create_pet_table manual__2021-12-13T20:41:00.066998+00:00 [queued]>
[2021-12-13 20:41:00,451] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: postgres_operator_dag.create_pet_table manual__2021-12-13T20:41:00.066998+00:00 [queued]>
[2021-12-13 20:41:00,451] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 20:41:00,451] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-13 20:41:00,451] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-13 20:41:00,465] {taskinstance.py:1262} INFO - Executing <Task(PostgresOperator): create_pet_table> on 2021-12-13 20:41:00.066998+00:00
[2021-12-13 20:41:00,470] {standard_task_runner.py:52} INFO - Started process 3597 to run task
[2021-12-13 20:41:00,474] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_operator_dag', 'create_pet_table', 'manual__2021-12-13T20:41:00.066998+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/example_postgres.py', '--cfg-path', '/tmp/tmp54l_en3w', '--error-file', '/tmp/tmpn61nv1ob']
[2021-12-13 20:41:00,475] {standard_task_runner.py:77} INFO - Job 14: Subtask create_pet_table
[2021-12-13 20:41:00,522] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_operator_dag.create_pet_table manual__2021-12-13T20:41:00.066998+00:00 [running]> on host ef07609851a1
[2021-12-13 20:41:00,576] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_operator_dag
AIRFLOW_CTX_TASK_ID=create_pet_table
AIRFLOW_CTX_EXECUTION_DATE=2021-12-13T20:41:00.066998+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-13T20:41:00.066998+00:00
[2021-12-13 20:41:00,588] {base.py:79} INFO - Using connection to: id: postgres_***_db. Host: postgres-***, Port: 5432, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-13 20:41:00,592] {dbapi.py:225} INFO - Running statement: -- create hello_world_table
DROP TABLE IF EXISTS pet;
CREATE TABLE pet (
    pet_id SERIAL PRIMARY KEY,
    name VARCHAR NOT NULL,
    pet_type VARCHAR NOT NULL,
    birth_date DATE NOT NULL,
    OWNER VARCHAR NOT NULL
    );, parameters: None
[2021-12-13 20:41:00,642] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=postgres_operator_dag, task_id=create_pet_table, execution_date=20211213T204100, start_date=20211213T204100, end_date=20211213T204100
[2021-12-13 20:41:00,687] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-13 20:41:00,732] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
